sim/src/image.rs

```diff
@@ -1528,7 +1528,11 @@ fn install_image(flash: &mut SimMultiFlash, slot: &SlotInfo, len: ImageSize,
                 // c::boot_status_sz(dev.align() as u32) as usize
                 16 + 4 * dev.align()
             } else {
-                c::boot_trailer_sz(dev.align() as u32) as usize
+                fn align_up(num: u32, align: u32) -> u32 {
```

Comment:
Probably should move this function to the top level. There are a handful of utility functions later in this file.
Also, probably best to have an assertion to make sure that 'align' is a power of two `assert!(align.is_power_of_two())`.

Reply:
Moved it into a dedicated file `utils.rs`.